core-rust/state-manager/src/store/rocks_db.rs

```diff
@@ -447,6 +447,32 @@ impl RocksDBStore {
         Ok(rocks_db_store)
     }
 
+    /// Creates a readonly [`RocksDBStore`] that allows reading from the store while some other
+    /// process is writing to it. Any write operation that happens against a read-only store leads
+    /// to a panic.
+    pub fn new_read_only(root: PathBuf) -> Result<RocksDBStore, DatabaseConfigValidationError> {
+        let mut db_opts = Options::default();
+        db_opts.create_if_missing(true);
+        db_opts.create_missing_column_families(true);
```

Comment:
I wonder how this 2x `true` really behave while `read_only` 🤔 

I suspect that regardless of `true` vs `false`, we would see a panic.
But with `false` this panic will be more accurate (i.e. "the logic expected [some CFs] but they don't exist in DB").

Reply:
Yeah, I think I'd rather these both be false I think?

Reply:
Looks like `false` make more sense here. I've made this change.